helm/cert-exporter/values.yaml

```diff
@@ -0,0 +1,6 @@
+namespace: monitoring
```

Comment:
I have a worry about this namespace. We only use `monitoring` in the control plane. For tenant clusters we set it to kube-system in the configmap created by cluster-operator.

https://github.com/giantswarm/cluster-operator/blob/f264ffc861ff304b1c87aca1a3b86091b1315ced/pkg/v21/configmap/desired.go#L136